### ISSUE TITLE
bug: Bunny Keeps Failing to install

### ISSUE THREAD

**Issue by SwiftCris** [pyoncord/Bunny#80]
**Describe the bug**
So, i click on "Install" from the manager 1.0.6 / and it wont downlaod
**To Reproduce**
1. Install "Bunny Manager"
2. Click on 'Install'
3. Wait a bit

**Expected behavior**
It should install and work

**Screenshots**
n/a
**Device Information (please complete the following information):**
 - Device: Samsung A23
 - OS: Android 14
 - Discord Version Latest
 - Bunny Version Latest

**Checklists**
- [x] I understand that this is a bug report and not a place to request new features or plugins
- [x] I understand that this is no place to report bugs produced by a plugin
- [x] I can reproduce this bug in Bunny's safe mode AND unable to reproduce it in vanilla/unmodded Discord

**Comment by Rico040** [comment]
can't you share logs?

**Comment by dwaaad** [comment]
I'm also having the same issue. Here's my logs:
`10-20-2024 11:45:15pm [I] Bunny Manager v1.0.6
10-20-2024 11:45:15pm [I] Built from commit 66026b4 on main (Changes Present)
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [I] Running Android 13, API level 33
10-20-2024 11:45:15pm [I] Supported ABIs: arm64-v8a, armeabi-v7a, armeabi
10-20-2024 11:45:15pm [I] Device: Xiaomi - 2201117TY (spesn)
10-20-2024 11:45:15pm [I] SOC: QTI SM6225
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [I] Adding Bunny to Discord v251.15 - Stable
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [I] Checking if base-251015.apk is cached
10-20-2024 11:45:15pm [I] base-251015.apk was not properly cached, downloading now
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [I] 
10-20-2024 11:45:15pm [E] Failed on DownloadBaseStep
10-20-2024 11:45:15pm [E] java.lang.IllegalArgumentException: Unknown URL content://downloads/my_downloads
10-20-2024 11:45:15pm [E] 	at android.content.ContentResolver.insert(ContentResolver.java:2192)
10-20-2024 11:45:15pm [E] 	at android.content.ContentResolver.insert(ContentResolver.java:2157)
10-20-2024 11:45:15pm [E] 	at android.app.DownloadManager.enqueue(DownloadManager.java:1668)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.domain.manager.DownloadManager.download(Unknown Source:138)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.installer.step.download.base.DownloadStep.run$suspendImpl(Unknown Source:326)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.installer.step.download.base.DownloadStep.run(Unknown Source:0)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.installer.step.Step.runCatching(Unknown Source:90)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.installer.step.StepRunner.runAll(Unknown Source:112)
10-20-2024 11:45:15pm [E] 	at dev.beefers.vendetta.manager.ui.viewmodel.installer.InstallerViewModel$job$1$1.invokeSuspend(Unknown Source:30)
10-20-2024 11:45:15pm [E] 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
10-20-2024 11:45:15pm [E] 	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:112)
10-20-2024 11:45:15pm [E] 	at androidx.work.Worker$2.run(Unknown Source:12)
10-20-2024 11:45:15pm [E] 	at kotlinx.coroutines.scheduling.TaskImpl.run(Unknown Source:2)
10-20-2024 11:45:15pm [E] 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:95)`

Every time I try to press '_Install_' the manager immediately fails to download the base APK. I've tried every mirror and many reinstalls of both Discord and Bunny Manager. I have also tried installing using Shizuku and as I've read in other bug reports here: my MIUI optimisation is disabled, but none of this has helped.
![Screenshot_2024-10-31-22-27-34-329_io.github.pyoncord.manager.jpg](https://github.com/user-attachments/assets/607e49b6-4ea0-44a1-80b4-5432de3e508b)

I'm running **Xiaomi HyperOS Android 13** on a **Redmi Note 11** and **latest versions** of both Discord and Bunny Manager. What's weird is my very first time installing Bunny went perfectly fine. But after installing too many plugins and realising I wanted the app to be faster I decided to just delete the entire thing and reinstall. Except I'm now running into this error every single time.
